cleanlab_studio/examples/python_api_demo.ipynb

```diff
@@ -0,0 +1,309 @@
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>index</th>\n",
+       "      <th>cleanlab_issue</th>\n",
+       "      <th>cleanlab_label_quality</th>\n",
+       "      <th>cleanlab_suggested_label</th>\n",
+       "      <th>cleanlab_clean_label</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>0</td>\n",
+       "      <td>False</td>\n",
+       "      <td>0.569078</td>\n",
+       "      <td>None</td>\n",
+       "      <td>None</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>1</td>\n",
+       "      <td>False</td>\n",
+       "      <td>0.736917</td>\n",
+       "      <td>None</td>\n",
+       "      <td>None</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>2</td>\n",
+       "      <td>True</td>\n",
+       "      <td>0.251255</td>\n",
+       "      <td>⭐⭐⭐</td>\n",
+       "      <td>None</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>3</td>\n",
+       "      <td>False</td>\n",
+       "      <td>0.556679</td>\n",
+       "      <td>None</td>\n",
+       "      <td>None</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>4</td>\n",
+       "      <td>True</td>\n",
+       "      <td>0.139512</td>\n",
+       "      <td>⭐⭐⭐</td>\n",
+       "      <td>None</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   index  cleanlab_issue  cleanlab_label_quality cleanlab_suggested_label   \n",
+       "0      0           False                0.569078                     None  \\\n",
+       "1      1           False                0.736917                     None   \n",
+       "2      2            True                0.251255                      ⭐⭐⭐   \n",
+       "3      3           False                0.556679                     None   \n",
+       "4      4            True                0.139512                      ⭐⭐⭐   \n",
+       "\n",
+       "  cleanlab_clean_label  \n",
+       "0                 None  \n",
+       "1                 None  \n",
+       "2                 None  \n",
+       "3                 None  \n",
+       "4                 None  "
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "cleanlab_columns_df = studio.download_cleanlab_columns(cleanset_id)\n",
+    "cleanlab_columns_df.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
```

Comment:
(not sure if i'm highlighting the right thing, but you have an empty cell at the bottom of the ipynb)